dev/com.ibm.ws.kernel.service/src/com/ibm/ws/common/crypto/CryptoUtils.java

```diff
@@ -139,7 +139,7 @@ public static String getCipherForAudit() {
     public static void logInsecureAlgorithm(String configProperty, String insecureAlgorithm) {
         // TODO remove beta check
         if (isRunningBetaMode()) {
-            Tr.warning(tc, "CRYPTO_INSECURE", configProperty, insecureAlgorithm);
+            Tr.warning(tc, "CRYPTO_INSECURE", configProperty, insecureAlgorithm, getSecureAlternative(insecureAlgorithm));
```

Comment:
getSecureAlternative() can return null if a suitable alternative isn't found. Do we want to guard against this?

Reply:
the places that are calling `logInsecureAlgorithm` are calling `isAlgorithmInsecure` first which only returns true if there's an alternative.

Reply:
Makes sense. LGTM